mod-tool/src/main/scala/at/forsyte/apalache/tla/tooling/opt/CheckCmd.scala

```diff
@@ -56,6 +56,10 @@ class CheckCmd(name: String = "check", description: String = "Check a TLA+ speci
     opt[String](name = "view", description = "the state view to use with --max-error=n, default: transition index",
         default = "")
 
+  var saveRuns: Boolean =
+    opt[Boolean](name = "save-runs", description = "save an example trace for each symbolic run, default: false",
+        default = false)
```

Comment:
I feel like the name "save-runs" isn't really correct, at least in the non-simulation case. Would it make sense to rename the flag in both cases to e.g. `enable-traces` or `tracing`? Also, the description here is not really suitable for `check` IMO ("each" is confusing here).

Reply:
> I feel like the name "save-runs" isn't really correct, at least in the non-simulation case

Why do you think that's the case? It's doing the same thing, right?

> Also, the description here is not really suitable for check IMO ("each" is confusing here).

Do you think "each" suggests that there are multiple?

I think Igor's idea was that we do not use the word "trace" with users.

Reply:
How about `--dump-traces`?

Reply:
Maybe `output-`, rather than `dump-`.

Reply:
Renamed in [63494ad](https://github.com/informalsystems/apalache/pull/2047/commits/63494ad4fffe6cba3443127ed9ea3566338d4c63)